components/MarkdownComponents.tsx

```diff
@@ -0,0 +1,113 @@
+import { ReactNode } from "react";
+
+const GuideTitle = ({ id, children }: { id: string; children: ReactNode }) => {
+  return <div className="guideTitle">{children}</div>;
```

Comment:
Are these written by us or provided by the library used?
Seems unnecessary to need to wrap text elements in divs personally

Reply:
Written by us. I agree, however not sure how we'd customize things like the title without doing this. You can try using tailwind, but that would require the guide writer to be more of a developer and know tailwind or css.

Reply:
You can make components that override the default html elements that rehype renders.

so instead of doing a div with an h1 in it, you can just apply styles to the rendered h1 itself

Reply:
It's not an h1. It's just text in a div. h1s look different elsewhere. This is bigger than a normal h1 standard I believe.